### DISCUSSION TITLE
n/a

### DISCUSSION THREAD

**Discussion by nlauer** [zkSync-Community-Hub/zksync-developers#611] [General]

**Comment by alhenaai** [comment]
It seems like you are experiencing a nonce-related issue when trying to send transactions from a specific address on the zkSync network. The error message "insertion of another transaction with the same nonce is in progress" indicates that there might be a transaction that is already in process or stuck in the mempool with the same nonce as the one you are trying to use.

Here are a few steps you can take to potentially resolve this issue:

1. **Check the Transaction Status**: Use the zkSync explorer to check if there are any pending transactions for your address that might be using the same nonce. If you find such a transaction, you will need to wait until it is processed or dropped from the mempool.

2. **Increment the Nonce Manually**: If the nonce you are using is indeed correct and unused, try manually incrementing the nonce to the next sequential number and send the transaction again. This might bypass the issue if the previous nonce is stuck.

3. **Reset or Reconfigure Your Wallet**: Sometimes, resetting your wallet or reconfiguring it can help clear stuck transactions or nonce issues. Make sure to back up your wallet before doing this.

4. **Use a Different RPC**: Although you mentioned that you have tried public and private RPCs, consider switching to another RPC endpoint if available, or ensure that the RPC you are using is fully synced and operational.

5. **Contact Support**: Since this issue persists and is specific to one address, it might be beneficial to reach out directly to zkSync support or the development team for more personalized assistance. They might be able to provide insights specific to your account or the internal workings of the network that are not visible externally.

If you need further assistance or have additional information to provide, feel free to reach out.

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Comment by bxpana** [comment]
Could you please share the address that you're having the issue with?

**Reply by bxpana** [reply]
What's your Discord name?

**Reply by bxpana** [reply]
Accepted the friend request and you can share the address there.
